README.md

```diff
@@ -1,183 +1,182 @@
-
-### Guidelines for writing code documentation
-
-- Rust follows specific style for documenting various code elements. Refer to [rust doc](https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html) and [documentation example](https://doc.rust-lang.org/rust-by-example/meta/doc.html) for more details.
-
-- Running ```RUSTDOCFLAGS="--enable-index-page -Zunstable-options" cargo doc --no-deps``` will generate documentation specific to Frequency while ignoring documenting dependencies.
-
-- To view generated cargo docs, one can open ```./target/doc/index.html```.
-
-### Generating a new spec file
-
-1. Update `node/chain_spec.rs` with required spec config, defaults to `para_id:2000` and relay chain to be `rococo_local.json` with `protocol_id:frequency-local`
-2. Run `cargo run --release build-spec --disable-default-bootnode > ./res/genesis/frequency-spec-rococo.json` to export the chain spec
-3. Run `cargo run --release build-spec --raw --disable-default-bootnode --chain ./res/genesis/frequency-spec-rococo.json > ./res/genesis/rococo-local-frequency-2000-raw.json` to export the raw chain spec
-4. Commit
-5. Alternatively, run ```./scripts/generate_specs.sh 2001 true``` to generate plain and raw frequency spec along with genesis state and wasm. Replace 2001 with registered parachain id.
-
-Note: To build spec against specific chain config; specify chain name in the command above.
-
-### Downloading Frequency related spec files, generated genesis state and wasm
-
-In order to experiment with Frequency, spec files and generated genesis state and wasm can be downloaded from [build artifacts](https://github.com/LibertyDSNP/frequency/actions/workflows/main.yml?query=branch%3Amain)
-
-## Linting
+   ```bash
+   make stop-relay
+   ```
+1. Run to remove unused volumes.
+   ```sh
+   docker volume prune
+   ```
+1. Clean up temporary directory to avoid any conflicts with next onboarding
+   ```sh
+   rm -fr /tmp/frequency
+   ```
+
+## Generate a New Spec File
+
+To build spec against specific chain config specify chain name in the command above.
+
+1. Update `node/**/chain_spec.rs` with required spec config, defaults to `para_id:2000`
+and relay chain to be `rococo_local.json` with `protocol_id:frequency-local`
+1. Export the chain spec
+   ```sh
+   cargo run --release build-spec --disable-default-bootnode > ./res/genesis/local/frequency-spec-rococo.json
+   ```
+1. Export the raw chain spec
+   ```sh
+   cargo run --release build-spec --raw --disable-default-bootnode --chain ./res/genesis/local/frequency-spec-rococo.json > ./res/genesis/local/rococo-local-frequency-2000-raw.json
+   ```
+
+~~Alternatively, run ```./scripts/generate_specs.sh 2001 true``` to generate plain and raw frequency spec along with genesis state and wasm. Replace 2001 with registered parachain id.~~
+
+Alternatively, run `make specs-local` to generate plain and raw frequency spec along with genesis state and wasm.
+
+# Lint
 
 - Lint the project with `cargo +nightly fmt`.
```

Comment:
it lints the rust code so you won't see any thing happening since all files are properly linted when we merge PR but anyone writing code generally lints and checks in to their PRs